tests/integration/testdata/start_api/swagger-rest-api-template.yaml

```diff
@@ -0,0 +1,102 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Globals:
+  Api:
+    BinaryMediaTypes:
+    - image~1png
+Resources:
+  Base64ResponseFunction:
+    Properties:
+      CodeUri: "."
+      Handler: main.base64_response
+      Runtime: python3.6
+    Type: AWS::Serverless::Function
+  EchoBase64EventBodyFunction:
+    Properties:
+      CodeUri: "."
+      Handler: main.echo_base64_event_body
+      Runtime: python3.6
+    Type: AWS::Serverless::Function
+  MyApi:
+    Properties:
+      Body:
```

Comment:
I plan to support both since the code is built into the SamSwaggerReader with FN::Transform for Body and adding the local file directly to BodyS3Location.

Reply:
So can `Body` have a value that is a path?

Reply:
I added a test to verify that Body can't have a local path.